ui/app/AppLayouts/Wallet/views/RightTabView.qml

```diff
@@ -227,11 +227,17 @@ RightTabBaseView {
                         onSwapRequested: root.launchSwapModal(key)
                         onReceiveRequested: root.launchShareAddressModal()
                         onCommunityClicked: Global.switchToCommunity(communityKey)
+
+                        onHideRequested: (key) => {
+                                             const token = ModelUtils.getByKey(model, "key", key)
+                                             Global.openConfirmHideAssetPopup(token.symbol, token.name, token.icon, !!token.communityId)
+                                         }
+                        onHideCommunityAssetsRequested: (communityKey) => Global.openPopup(confirmHideCommunityAssetsPopup, { communityId: communityKey })
```

Comment:
Required parameters `name` and `icon` seems to be not provided to the popup.

I would also just create it simply via `createObject`. `Global.openPopup` is using cache which makes not too much sense in this case.

Reply:
Fixed